### PR TITLE
[15.0][FIX] edi_voxel*: remove job decorator from delayable methods

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @pedrobaeza

File: edi_voxel_account_invoice_oca/data/queue_job_function_data.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- Copyright 2023 Tecnativa - Ernesto tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="job_function_get_and_send_voxel_report" model="queue.job.function">
```
Comment (pedrobaeza) [inline edi_voxel_account_invoice_oca/data/queue_job_function_data.xml:4]: Shouldn't this requires a migration script to find the existing functions and channels and put them the XML-IDs?

Comment (pedrobaeza) [reply edi_voxel_account_invoice_oca/data/queue_job_function_data.xml:4]: If not, I think there will be unique constraint errors.

**Comment by ernestotejeda** [comment]
I changed the development status of stock_picking_invoice_link https://github.com/OCA/stock-logistics-workflow/pull/1181

**Comment by ernestotejeda** [comment]
@pedrobaeza I added the migration script, review it, please. thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**